src/machine/board_wioterminal.go

```diff
@@ -349,8 +349,11 @@ var (
 
 	UART1 = &sercomUSART2
 
-	// RTL8720D
+	// RTL8720D (tx: PC22, rx: PC23)
```

Comment:
@sago35 should this be removed? Or it is possible to use UART2 with the RTL8720D?

Reply:
@deadprogram 

It is possible to use UART2 with the RTL8720D.

These ports are not required for networking, but are needed for rewriting and debugging.
Probably most people will never handle it, but driver developers may need it from time to time.

![image](https://user-images.githubusercontent.com/9251039/187884422-6608cf4a-475d-4ed5-b325-08aec556328b.png)

Reply:
Thank you very much for the incredibly clear explanation!